app/services/editions/publishing/EditionsBucket.scala

```diff
@@ -11,14 +11,15 @@ class EditionsBucket(s3Client: AmazonS3, bucketName: String) {
   def createIssuePrefix(issue: PublishedIssue): String = s"${issue.name}/${issue.issueDate.toString}"
 
   def createIssueFilename(issue: PublishedIssue): String = {
-    val keyname = issue.version.getOrElse("preview")
+    val keyname = issue.publicationEventId.getOrElse("preview")
     s"$keyname.json"
   }
 
   def putIssue(issue: PublishedIssue) = {
     val issueJson = Json.stringify(Json.toJson(issue))
     val metadata = new ObjectMetadata()
     metadata.setContentType("application/json")
+    issue.publicationEventId.foreach(metadata.addUserMetadata("publication-event-id", _))
     val key = s"${createIssuePrefix(issue)}/${createIssueFilename(issue)}"
     val request = new PutObjectRequest(bucketName, key, new StringInputStream(issueJson), metadata)
```

Comment:
i think for custom metadata in s3 there is a convention to start with `x-`: https://docs.aws.amazon.com/AmazonS3/latest/user-guide/add-object-metadata.html